### DISCUSSION TITLE
Actividad_0 - Chesini Pablo

### DISCUSSION THREAD

**Discussion by pchesini** [frlp-utn-ingsoft/catedra#34] [Show and tell]
https://github.com/pchesini/Actividad_0.git

**Comment by RodrigoJacznik** [comment]
Hola.
Los commits están bien. La app funciona correctamente, lo único para mencionar es que en el template de detalle de una `poll` te quedo un `for` de mas, si queres lo podes arreglar (si fue intencional déjalo así).
La actividad esta aprobada.

Saludos.

**Comment by pchesini** [comment]
Listo, ahi corregi el for redundante, lo que no sabia era como comitear ese cambio, como el comit estaba en el 4.2 , la modicficacion quedo en el 4.2.1.
